EasyPost.Net35/Tuple.cs

```diff
@@ -1,13 +1,17 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿// Tuple.cs
+// Copyright (c) 2022 EasyPost
+// All rights reserved.
 
-namespace EasyPost {
-    public class Tuple<T1, T2> {
-        public T1 Item1 { get; private set; }
-        public T2 Item2 { get; private set; }
-        internal Tuple(T1 first, T2 second) {
+namespace EasyPost.Net35
+{
+    public class Tuple<T1, T2>
+    {
+        public T1 Item1 { get; }
```

Comment:
I guess the auto-formatted removed it because it wasn't being used anywhere. I'll patch that.

Reply:
Actually, this whole class appears to be unused. Was only added for the 3.5 library. Removed in cb1a601, awaiting test results on 3.5 to see if it can be removed permanently.

Reply:
All tests passed with this class removed: https://github.com/EasyPost/easypost-csharp/runs/4973970593?check_suite_focus=true

Reply:
Cool, was this removed completely then? I don't see any references to this anywhere some maybe it's orphaned.

Reply:
I have removed it, yes